morpheushelper/morpheushelper.py

```diff
@@ -334,13 +340,13 @@ async def on_bot_ping(message: Message):
 register_cogs(bot, *enabled_cogs)
 
 if bot.cogs:
-    print(f"\033[1m\033[32m{len(bot.cogs)} Cog{'s' * (len(bot.cogs) > 1)} enabled:\033[0m")
+    root_logger.info("%s Cog%s enabled:", len(bot.cogs), 's' * (len(bot.cogs) > 1))
```

Comment:
maybe it would be more readable if we use if else instead of the * operator
```python
root_logger.info("%s Cog%s enabled:", len(bot.cogs), 's' if len(bot.cogs) > 1 else '')
```